tests/ui/runtime/stdout-before-main.run.stdout

```diff
@@ -0,0 +1 @@
+Hello from before main!
```

Comment:
Could the call in `main` be changed to `println` so this file doesn't wind up without a newline?

Reply:
Sure!